app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,18 @@
+module StaticPagesHelper
+  def homepageContent
+    [
+      {
+        title: 'What is the purpose of the project',
```

Comment:
Should we move this to the locales? Just thinking it would make translating the rest of the site easier in the future, and helpers probably aren't the best place to store data